src/Ideal.jl

```diff
@@ -15,6 +18,27 @@ function *(R::Ring, x::RingElement)
   return ideal(R, x)
 end
 
+function *(R::Ring, x::Any)
+  return ideal(R, R(x))
```

Comment:
Hmm, but won't that mean that e.g. `R * big(2)` might work but not `R * ZZRingElem(2)` ?

Might be less confusing to just keep the new methods and remove those in line 13-19 ?

Reply:
You are of course right. Fixed this by keeping only the new methods.